package.json

```diff
@@ -72,6 +73,7 @@
     "@typescript-eslint/parser": "4.14.1",
     "buffer": "6.0.1",
     "cache-loader": "4.1.0",
+    "codecov": "^3.8.2",
```

Comment:
@travis-payne Please lock down the npm versions to exact ones. This is so that we can have reproducible builds: https://reproducible-builds.org/

Reply:
Done!